msal/tests/Test.MSAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -69,7 +69,8 @@ public void InitializeBeforeTest()
         [Test]
         public void AcquireTokenTest()
         {
-            _msalMobileTestHelper.AcquireTokenInteractiveTestHelper(xamarinController, LabUserHelper.GetUser(LabUserHelper.DefaultUserQuery));
+            var labResponse = LabUserHelper.GetLabUserAndApp(LabUserHelper.DefaultUserQuery);
+            _msalMobileTestHelper.AcquireTokenInteractiveTestHelper(xamarinController, labResponse);
```

Comment:
can we change this to LabUserHelper.GetLabUserAndApp(LabUserHelper.DefaultUserQuery) like the other functions?

Reply:
When people first try out to run automation, they will get into problems because installing that certificate is a pain and IWA doesn't work because of MFA. I will change it, but it's not a bad idea to separate the 2.